README.md

```
@@ -1,66 +1,96 @@
-This rule is fixable. The fixer will automatically move `null` and `undefined` to the end of the union type.
-
-## Contributing
-
-If you want to contribute to this project, feel free to submit issues or pull requests.
-
-## License
-
-This project is licensed under the MIT License.
+## Rules
+
+### `yenz/type-ordering`
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)